src/antic/nf_elem.jl

```diff
@@ -1178,22 +1157,23 @@ rand(K::AnticNumberField, r) = rand(Random.GLOBAL_RNG, K, r)
 ###############################################################################
 
 @doc Markdown.doc"""
-    NumberField(f::fmpq_poly, s::Union{AbstractString, Char, Symbol};
+    number_Field(f::fmpq_poly, s::Union{AbstractString, Char, Symbol};
                 cached::Bool = true, check::Bool = true)
 
 Return a tuple $R, x$ consisting of the parent object $R$ and generator $x$
 of the number field $\mathbb{Q}[x]/(f)$ where $f$ is the supplied polynomial.
 The supplied string `s` specifies how the generator of the number field
-should be printed.
+should be printed. If `s` is not specified, it defaults to `_a`.
 """
-function NumberField(f::fmpq_poly, s::Union{AbstractString, Char, Symbol}; cached::Bool = true, check::Bool = true)
+function number_field(f::fmpq_poly, s::Union{AbstractString, Char, Symbol}="_a",; cached::Bool = true, check::Bool = true)
```

Comment:
Maybe use a symbol to avoid a pointless conversion from String to Symbol?
```suggestion
function number_field(f::fmpq_poly, s::Union{AbstractString, Char, Symbol}=:_a; cached::Bool = true, check::Bool = true)
```